pkg/azurediskplugin/Dockerfile

```diff
@@ -14,7 +14,7 @@
 
 ARG ARCH
 FROM k8s.gcr.io/build-image/debian-base:buster-v1.6.0
-RUN clean-install util-linux e2fsprogs mount ca-certificates udev xfsprogs
+RUN clean-install util-linux e2fsprogs mount ca-certificates udev xfsprogs telnet
```

Comment:
Why do we now need telnet?

Reply:
Ah this was change I made with the previous design of clean up. It is no longer needed; I will remove it.